config/defaults.json

```diff
@@ -2,7 +2,7 @@
   "vault": {
     "port": 8200,
     "hostname": "localhost",
-    "token": "3e5f8293-eb83-37be-2ed3-03e4324c5e53"
+    "token":
```

Comment:
I added a string there to make it obvious where to add the token string. This should make it easier to write the getting started guide as well.